modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/change-mdf-listing-pages/index.js

```diff
@@ -18,6 +18,24 @@ const siteURL = Liferay.ThemeDisplay.getLayoutRelativeURL()
 const buttonMDFRequest = fragmentElement.querySelector('#mdf-request');
 const buttonMDFClaim = fragmentElement.querySelector('#mdf-claim');
 
+const queryParams = new URLSearchParams(window.location.search);
+const newSuccess = queryParams.get('new-success');
+const editSuccess = queryParams.get('edit-success');
```

Comment:
```suggestion
const hasNewSuccessParam = Boolean(queryParams.get('new-success'));
const hasEditSuccessParam = Boolean(queryParams.get('edit-success'));
```

Pay attention for naming convention used here.
It should be more clear as possible.

Also, it good to add `Boolean` that means we are check if the value exists. In other words, we are change the value to a boolean type.

ref.: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean#boolean_coercion